docs/howto/support/raise-issues.md

```diff
@@ -0,0 +1,124 @@
+Generally speaking one could say we only have two issue types, Service Requests and Incidents.
+These are two very different issue types and it is important that we understand in what ways.
+
+#### Incidents
+
+When you issue a incident ticket with us you are reporting an **ongoing** incident.
+An incident is something that is hindering your normal operation, when an issue with our services is impacting your systems, customers or users.
+
+#### Service Requests
+
+A Service Request is the appropriate issue to create for anything that is *not* an incident.
+Some examples are:
+
+- Request for quota increase
+- Request for information
+- Invoice issues
+- Account issues
+- Cloud Management Portal issues
+
+### Priority Matrix
+
+We apply the priority matrix when we set our priorities.
+It gives us the ability to prioritize incidents, requests and changes.
+With the matrix we balance the impact and urgency of the issue tickets so that we can assign, communicate, and resolve accordingly and appropriately.
+
+The way we apply this is that you, the customer, assigns a **Priority** based on the priority matrix of **Urgency** and **Impact**.
+You can see the priority matrix below.
+
+|                | High impact | Medium impact | Low impact |
+|---------------:|:-----------:|:-------------:|:----------:|
+|   High urgency |      P1     |       P2      |     P3     |
+| Medium urgency |      P2     |       P2      |     P3     |
+|    Low urgency |      P3     |       P3      |     P4     |
+
+---
+
+## Email Issue Ticket
+
+When you send us an email to raise an issue ticket, please keep the following in mind:
+
+- Send your email to [{{support_email}}](mailto:{{support_email}}), from an email address [matching your account data](../account-billing/change-account-data.md).
+- Use English.
+  Some of our personnel do speak Swedish and other languages, but issuing tickets in any other language than English may result in longer response time.
+- Use a descriptive subject, like "Request to raise project quota" or "Unable to login to {{gui}}".
+- Include as much information as possible.
+  In particular, detail what you've already tried resolve the issue.
+  Also, if applicable, include the UUID (Universally unique identifier) of the relevant resource(s).
+
+Please note that if you have a Support Package subscription, we require you to issue your support tickets through the {{support}} portal.
+
+---
+
+## Support Portal Ticket
+
+In the {{support}} portal there are three categories of request types, each with several issue types.
+
+The ticket menu works in a dynamic way which means that depending on your selections made throughout the menu you will have different optional and required fields where you can input information.
+Always try to include as much relevant information as possible, this will help us help you more efficiently.
+
+1. Go to [{{support}} portal](https://{{support_domain}}).
```

Comment:
Do we want this link to point to the root of the domain, or to the `/servicedesk` path? (It's currently the latter everywhere else in the documentation.)

Reply:
Ah! It should be the servicedesk link, I must have missed that detail. Good eyes!

Reply:
No worries, fixed. :)